docs/ibm-granite.md

```diff
@@ -0,0 +1,87 @@
+# 🐝 IBM Granite Bee
+
+The [IBM Granite](https://www.ibm.com/granite) family of models can be used as the underlying LLM within Bee Agents. Granite™ is IBM's family of open, performant, and trusted AI models tailored for business and optimized to scale your AI applications.
+
+This guide and the associated [ibm_granite_bee](/examples/agents/ibm_granite_bee.ts) example will help you get started with creating Bee Agents using Granite.
+
+## 📦 Prerequisites
+
+### LLM Services
+
+IBM Granite is supported by [watsonx.ai](https://www.ibm.com/products/watsonx-ai) and [Ollama](https://ollama.com/). Watsonx.ai will allow you to run models in the cloud. Ollama will allow you to download and run models locally.
+
+> [!TIP]
+> Better performance will be achieved by using larger Granite models.
+
+> [!NOTE]
+> If you work for IBM there are additional options to run IBM Granite models with VLLM or RITS.
+
+#### Ollama
+
+There are guides available for running Granite with Ollama on [Linux](https://www.ibm.com/granite/docs/run/granite-on-linux/granite/), [Mac](https://www.ibm.com/granite/docs/run/granite-on-mac/granite/) or [Windows](https://www.ibm.com/granite/docs/run/granite-on-windows/granite/).
+
+#### Watsonx
+
+In order to use IBM [watsonx.ai](https://www.ibm.com/products/watsonx-ai) you can follow the [IBM watsonx as a Service](https://www.ibm.com/docs/en/watsonx/saas) documentation and in particular their [getting started tutorials](https://www.ibm.com/docs/en/watsonx/saas?topic=getting-started-tutorials).
+
+## 🛠️ Getting Started
+
+Before getting started, ensure you have access to an IBM Granite model and have followed any other requirements in the [prerequisites](#prerequisites) above.
+
+The [ibm_granite_bee](/examples/agents/ibm_granite_bee.ts) example agent is set up to demonstrate how to use IBM Granite with the Bee Agent Framework. In order to run this example, take the following steps:
+
+> [!TIP]
+> More detailed installation instructions are available for step 1 in the [Location Installation](https://github.com/i-am-bee/bee-agent-framework?tab=readme-ov-file#local-installation) section of our main README
+
+1. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) the Bee Agent Framework git repository by running the following command:
+
+   ```shell
+   git clone https://github.com/i-am-bee/bee-agent-framework.git
+   yarn install --immutable
+   ```
+
+1. Copy the [.env.template](/.env.template) file to a new file called `.env`
+
+1. Fill out the required details within the `.env` file, as follows
+
+   1. For watsonx (replace the values shown in braces):
+
+      ```ini
```

Comment:
I think it should be ```.env

Reply:
Resolved in [0b85aab](https://github.com/grahamwhiteuk/bee-agent-framework/commit/0b85aabf1dfcabe29bdf9dc6529328c8d74e9afc)